js/src/scraper-store.js

```diff
@@ -8,6 +8,7 @@ var scraperObjects = {
     'textarea':     require( "./scraper/scraper.textarea.js" ),
     'email':        require( "./scraper/scraper.email.js" ),
     'url':          require( "./scraper/scraper.url.js" ),
+    'link':          require( "./scraper/scraper.link.js" ),
```

Comment:
Can you remove a single space before `require` to have it aligned with the other requires